backend/authentication-microservice/src/main/java/com/ss/utopia/auth/controller/SessionController.java

```diff
@@ -90,10 +94,12 @@ public User getSession(@AuthenticationPrincipal UserDetails currentUser) {
       return userService.getUserByUsername(currentUser.getUsername());
   }
 
+  // TODO: 2/6/2021 POST /api/sessions/users/user/logout
   @DeleteMapping
   @ResponseStatus(HttpStatus.NO_CONTENT)
   public void logout(HttpServletResponse response) {
-    final Cookie sessionCookie = userService.removeCookie();
+    final Cookie sessionCookie = userService.removeCookie()
```

Comment:
Why are we throwing and error? Doesn't removeCookie just return a new Cookie object with a hard coded value and path? When would that ever throw an error?
See [constructor](https://docs.oracle.com/javaee/6/api/javax/servlet/http/Cookie.html#Cookie(java.lang.String,%20java.lang.String)) and [setPath](https://docs.oracle.com/javaee/6/api/javax/servlet/http/Cookie.html#setPath(java.lang.String)) documentation.

Reply:
Added per Patrick request.